flixel/util/FlxDirection.hx

```diff
@@ -0,0 +1,50 @@
+package flixel.util;
+
+@:enum abstract FlxDirection(Int) from Int to Int
+{
+	var LEFT    = 0x0001;
+	var RIGHT   = 0x0010;
+	var UP      = 0x0100;
+	var DOWN    = 0x1000;
+	var NONE    = 0x0000;
+	var CEILING = 0x0100;// UP;
```

Comment:
These should be doc comments, not line comments, so they show up in API docs and completion.

Would also make sense to have a line comment with a link to the relevant Haxe issue once there is one.

Reply:
https://github.com/HaxeFoundation/haxe/issues/10237